packages/node_modules/@webex/redux-module-spaces/src/actions.js

```diff
@@ -117,14 +117,14 @@ function storeInitialSpace(id) {
 /**
  * Adds a rate limit error to the errors module
  * @param {function} dispatch
- * @param {String} code
+ * @param {String} name
  */
-function addLoadError(dispatch, code) {
+function addLoadError(dispatch, name) {
   dispatch(addError({
-    code,
+    code: name,
     id: 'redux-module-spaces-load',
     displayTitle: 'Something\'s not right',
-    displaySubtitle: 'Unable to load spaces. Please try again later.',
+    displaySubtitle: `Unable to load spaces. Please try again later. [${name}]`,
```

Comment:
Should the error be on a different line? or have a nicer message of the error?

Reply:
Potentially, this means we'll be injecting formatting into a string that did not have formatting originally, resulting in a coupling of business logic with presentation logic.

We could create a larger effort to fix the presentation of all error actions within the redux store.